udata/core/dataset/models.py

```diff
@@ -335,7 +335,7 @@ def to_mongo(self, *args, **kwargs):
 
 
 class ResourceMixin(object):
-    id = db.AutoUUIDField(primary_key=True)
+    id = db.AutoUUIDField(primary_key=True, unique=True)
```

Comment:
Do we need the manual check if we already have `unique=True`?

Reply:
Yes I can remove this, I didn't find a way to make it work

Reply:
I removed it. Adding it fails a lot of tests I don't know why… https://app.circleci.com/pipelines/github/opendatateam/udata/5863/workflows/ee92a1f9-786c-42dc-a1e9-dc5b5a3be992/jobs/33495?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-checks-link&utm_content=summary